_data/api-tools.yml

```diff
@@ -25,6 +34,24 @@
   runtime: "https://img.shields.io/badge/API_Runtime-No-red.svg"
   testing: "https://img.shields.io/badge/API_Testing-Yes-brightgreen.svg"
   notes: ""
+- name: "BurpSuite Professional"
+  link: "https://portswigger.net/burp/pro"
+  from: "PortSwigger"
+  license: "https://img.shields.io/badge/license-Commercial-9cf.svg"
+  platforms: "https://img.shields.io/badge/Platform-Cross_Platform-inactive.svg"
+  posture: "https://img.shields.io/badge/API_Posture-No-red.svg"
+  runtime: "https://img.shields.io/badge/API_Runtime-No-red.svg"
+  testing: "https://img.shields.io/badge/API_Testing-Yes-brightgreen.svg"
+  notes: "<br/><em>Note: See <a href='https://portswigger.net/support/using-burp-to-test-a-rest-api'>Zap FAQ</a> for testing APIs</em>"
```

Comment:
I’m tempted to leave the zap reference just because it’s hilarious, but you should probably tweak that

Reply:
That has to be one the most hilariously ironic copy/paste fails of all time. 

Thanks for ignoring the call of the dark site.  Updating now...